### ISSUE TITLE
Label within qr code image

### ISSUE THREAD

**Issue by sumanc** [alexeyten/qr-image#54]
Hi,

Is there any support to add a label in the middle of the qr image?

Thanks,

--Suman

**Comment by sudonitin** [comment]
@sumanc by any chance did you get a resolution for this?

**Comment by sumanc** [comment]
Not really. We ended up settling to have it at the bottom which worked out okay.

**Comment by sudonitin** [comment]
and how'd you do it? Could you please let us know about your approach?

**Comment by sumanc** [comment]
There is no way to use this library to generate the QR image with text embedded. Ours is a browser app and we used this to generate the image and placed in a table cell, and placed the actual text in a different cell right underneath it.

**Comment by sudonitin** [comment]
ohh okay, I was able to figure it out using another dependency i.e. sharp

It came with a tradeoff with the latency being up by 2-3 times.


![1111](https://user-images.githubusercontent.com/29048183/209273607-2fa98e5e-596f-496f-9b46-073af94f05a4.png)
